src/Components/test/E2ETest/ServerExecutionTests/ServerAuthTest.cs

```diff
@@ -62,7 +62,7 @@ private void PerformReconnection()
             ((IJavaScriptExecutor)Browser).ExecuteScript("Blazor._internal.forceCloseConnection()");
 
             // Wait until the reconnection dialog has been shown but is now hidden
-            new WebDriverWait(Browser, TimeSpan.FromSeconds(10))
+            new WebDriverWait(Browser, TimeSpan.FromSeconds(20))
```

Comment:
@barahonajm do you think it would help if we performed a retry almost immediately after a disconnect was detected? It'd fairly similar to the previous behavior and we wouldn't need to change all these tests

Reply:
Yes, that sounds reasonable for me I am going to do the change

Reply:
although last time I consider this my solutions where:
* Move the `wait` method to the end of the reconnect process but now there is no reason to wait on the final try hence we need to add another try to match the time period the server keeps the connection

* Change e first try to wait `3000` ms like it was before and continue with the configured `retryIntervalMilliseconds` for the following tries 

@pranavkm could you help me pointing what of those options will be an acceptable approach?

Reply:
How about option 2?

Reply:
Options 2 (change the first try to `3000` unless we are no talking of the same options) works well although the user may feel their settings it not being considering at the beginning

Reply:
I was about to commit but forget to check jest test so to don't impact them (because they define 5ms as interval time) the interval calculation is as follow:

With `static readonly MaximumFirstRetryInterval = 3000`  

```js
const delayDuration = i == 0 && options.retryIntervalMilliseconds > ReconnectionProcess.MaximumFirstRetryInterval 
                      ? ReconnectionProcess.MaximumFirstRetryInterval 
                      : options.retryIntervalMilliseconds
```

If that is ok for you I will commit the changes

Reply:
Sure, that seems fine.

Reply:
done